apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/test/java/gov/cms/bfd/pipeline/ccw/rif/load/RifLoaderIT.java

```diff
@@ -687,7 +687,6 @@ public void loadSyntheticData() {
     verifyRecordPrimaryKeysPresent(samples);
   }
 
-  @Disabled
   @Test
```

Comment:
can you add a comment on this method as javadoc to explain the purpose of the test?

Reply:
Done